plugin/src/main/java/com/denizenscript/denizen/events/block/BrewsScriptEvent.java

```diff
@@ -48,6 +55,16 @@ public ObjectTag getContext(String name) {
         switch (name) {
             case "location": return location;
             case "inventory": return InventoryTag.mirrorBukkitInventory(event.getContents());
+            case "fuel_level": return new ElementTag(event.getFuelLevel());
+            case "result":
+                ListTag results = new ListTag();
+                for (ItemStack itemStack : event.getResults()){
+                    if (itemStack == null){
+                        results.addObject(new ItemTag(Material.AIR));
+                    }
+                    results.addObject(new ItemTag(itemStack));
+                }
+                return results;

```

Comment:
Can use the ListTag convertor constructor - `new ListTag(getResults, ItemTag::new)`